spacy/language.py

```diff
@@ -1649,7 +1653,7 @@ def _fix_pretrained_vectors_name(nlp: Language) -> None:
     else:
         raise ValueError(Errors.E092)
     for name, proc in nlp.pipeline:
-        if not hasattr(proc, "cfg"):
+        if not hasattr(proc, "cfg") or not isinstance(proc.cfg, dict):
```

Comment:
This `isinstance` check is harder to replace, but I definitely think we should.

Reply:
Wait do we still need this function?

Reply:
I'm not sure? 

In general, we definitely need to make sure `Pipe.cfg` is a dict, though, because we make assumptions about it elsewhere.